app/models/manifest.rb

```diff
@@ -50,6 +50,9 @@ def designated_facility_name
     content_field('designated_facility.name')
   end
 
+  def international_shipment
+  end
```

Comment:
when the form renders on a failed submit, Rails tries to autofill form fields based on database attributes / model methods. since we have a boolean for `international_shipment`, it gets a an `undefined method` error on render without this.

there is a larger issue with this form, which is that right now *some* field values are retained on render, but not all because of our unusual database setup. going to create a card for this because it would be possible to retain all values for the `content` on a `render :new` if we defined methods for each one.